qa/multi_input/multi_input_client.py

```diff
@@ -90,31 +90,37 @@ def main():
     # Infer
     outputs = []
     input_names = ["DALI_X_INPUT", "DALI_Y_INPUT"]
-    output_names = ["DALI_OUTPUT_X", "DALI_OUTPUT_Y"]
+    output_names = ["DALI_unchanged", "DALI_changed"]
+
     input_shape = list(input_data.shape)
+    multiplier = 2
+
     for oname in output_names:
         outputs.append(tritonclient.grpc.InferRequestedOutput(oname))
 
     for batch in batcher(input_data, FLAGS.batch_size):
         print("Input mean before backend processing:", np.mean(batch))
+        batch_size = np.shape(batch)[0]
+        print("Batch size: ", batch_size)
+
         # Initialize the data
-        input_shape[0] = np.shape(batch)[0]
-        print("Batch size: ", input_shape[0])
-        inputs = [tritonclient.grpc.InferInput(iname, input_shape, "UINT8") for iname in input_names]
+        input_shape[0] = batch_size
+        inputs = [tritonclient.grpc.InferInput(iname, input_shape, "UINT8") for iname in
+                  input_names]
         for inp in inputs:
             inp.set_data_from_numpy(np.copy(batch))
 
         # Test with outputs
-        results = triton_client.infer(model_name=model_name,
-                                      inputs=inputs,
-                                      outputs=outputs)
+        results = triton_client.infer(model_name=model_name, inputs=inputs, outputs=outputs)
 
         # Get the output arrays from the results
         for oname in output_names:
+            print("\nOutput: ", oname)
             output0_data = results.as_numpy(oname)
             print("Output mean after backend processing:", np.mean(output0_data))
             print("Output shape: ", np.shape(output0_data))
-            if not math.isclose(np.mean(output0_data), np.mean(batch)):
+            expected = np.mean(batch) * (1 if oname is "DALI_unchanged" else multiplier)
+            if not math.isclose(np.mean(output0_data), expected):
```

Comment:
why do we compare only the mean?
Couldn't be it smth like:
```python
expected = batch * (1 if oname is "DALI_unchanged" else multiplier)
if not np.allclose(output0_data, expected): 
   (...)
```

Reply:
Done